### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by yann9278**

### CODE REVIEW

**Review by pelegrand — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-      <div className="rsw-item">
-        <p>
-          Start in the <code>App.jsx</code> component !
-        </p>
+      <div>
+        <div>
+          <Switch>
+            <Route exact path="/" component={Home} />
+            <Route path="/Sport" component={Sport} />
+            <Route path="/BienManger" component={BienManger} />
```
Comment (tyZulh) [inline src/App.jsx:24]: Components name must be in english

File: src/App.jsx [hunk 2]
```diff
@@ -1,27 +1,33 @@
 import React from 'react';
```
Comment (tyZulh) [inline src/App.jsx:1]: For the import order :

```suggestion
import React from 'react';
import { Switch, Route } from 'react-router-dom';

import NavBarr from './components/NavBarr';
import BienManger from './components/BienManger';
import Sport from './components/Sport';
import BienDansSaTete from './components/BienDansSaTete';
import Footer from './components/Footer';
import Home from './components/Home';

import './App.css';
```

File: src/components/BienDansSaTete.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import './BienDansSaTete.css';
```
Comment (tyZulh) [inline src/components/BienDansSaTete.jsx:1]: Import order : 

React dependencies

Third party dependencies

Components

Style elements

File: src/components/BienManger.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useState } from 'react';
+import './BienManger.css';
```
Comment (tyZulh) [inline src/components/BienManger.jsx:3]: Import order :

React dependencies

Third party dependencies

Components

Style elements

File: src/components/Sport.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useState } from 'react';
```
Comment (tyZulh) [inline src/components/Sport.jsx:2]: Import order :

React dependencies

Third party dependencies

Components

Style elements

**Review by tyZulh — CHANGES_REQUESTED** [review]
Your components  "ShowRecipesFrigo" and "ShowRecipes" are identical, you don't need both

**Pull request closed without merging**